tests/python/gaia-ui-tests/gaiatest/apps/ui_tests_privileged/regions/user_media.py

```diff
@@ -14,14 +14,20 @@ class UserMediaPage(Base):
     _frame_locator = (By.CSS_SELECTOR, "#test-iframe[src*='getusermedia']")
 
     def switch_to_frame(self):
-        self.wait_for_element_displayed(*self._frame_locator)
-        get_user_media_frame = self.marionette.find_element(*self._frame_locator)
-        self.marionette.switch_to_frame(get_user_media_frame)
+        self.wait_for_element_displ
```

Comment:
This is obviously not correct :)
This caused failures in test_privileged_app_audio_capture_prompt.py and test_privileged_app_video_capture_prompt.py for me locally